src/test/java/org/eclipse/uprotocol/ulink/uTestApp.java

```diff
@@ -0,0 +1,50 @@
+package org.eclipse.uprotocol.ulink;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+import java.util.concurrent.Executors;
+
+import org.eclipse.uprotocol.cloudevent.datamodel.UCloudEventAttributes;
+import org.eclipse.uprotocol.uri.datamodel.UAuthority;
+import org.eclipse.uprotocol.uri.datamodel.UEntity;
+import org.eclipse.uprotocol.uri.datamodel.UResource;
+import org.eclipse.uprotocol.uri.datamodel.UUri;
+import com.google.protobuf.Any;
+
+public class uTestApp implements EventListener {
```

Comment:
uTestApp is an EventListener that can only listen on **one** topic since the signature is
public void onEvent(UUri **topic**, Any data, UCloudEventAttributes attributes) {

What happens if my app needs to listen on a couple of topics - merge the data and act accordingly
A simple example is listening for events from different vehicles - they are separate topics